src/sausage-rolls.md

```diff
@@ -0,0 +1,25 @@
+# Sausage Rolls
+
+- ⏲️ Prep time: 10 min
+- 🍳 Cook time: 30-60 min
+
+## Ingredients
+
+- Eggs
+- Puff pastry
+- Pork meat / sausage meat
```

Comment:
Would be useful to have some measurements or amounts here. ie, how many eggs, how much meat etc.